File: HASKELL/1006.hs
import Text.Printf
main::IO()
main = do
 a <- readLn ::IO Double
 b <- readLn ::IO Double
 c <- readLn ::IO Double
 let m =  (a*2.0 + b*3.0 + c*5.0)/10.0
 printf "MEDIA = %.1f\n" $ m

File: KOTLIN/1006.kt
import java.util.Scanner
 class Peso
{
    companion object{
        val a: Int = 2
        val b: Int = 3
        val c: Int = 5
    }
}

fun main(args:Array<String>){
    val sc = Scanner(System.`in`)
    
    var a : String = sc.next()
    var b : String = sc.next()
    var c : String = sc.next()

    println(saida(media(a.toDouble(),b.toDouble(),c.toDouble())))
}

fun saida(vlr:Double):String = "MEDIA = %.1f".format(vlr)
fun media(a:Double, b:Double, c:Double):Double =  (a * Peso.a + b * Peso.b + c*Peso.c) / 10.0


